src/workloads/filesystem/Flushing.yml

```diff
@@ -0,0 +1,21 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-perf"
+Description: >
+  Run large insertMany's to stress checkpointing, journal flush and dirty memory with {w:majority}.
+  The important ftdc metrics to look at are:
```

Comment:
Could you add something saying the documents are simple and there are no indexes?